go.mod

```diff
@@ -60,6 +59,7 @@ require (
 	github.com/emicklei/go-restful v2.9.5+incompatible // indirect
 	github.com/evanphx/json-patch v4.11.0+incompatible // indirect
 	github.com/felixge/httpsnoop v1.0.1 // indirect
+	github.com/go-logr/logr v0.4.0 // indirect
```

Comment:
should we need the indirect reference

Reply:
unfortunately, we can't. looks like an imported package is using this, so `go mod tidy` will automatically add it.